tests/check/check_h26xsigned_sign.c

```diff
@@ -673,6 +673,49 @@ START_TEST(two_completed_seis_pending)
 }
 END_TEST
 
+/* Test description
+ * Verify that after golden SEI is generated
+ * The operation is as follows:
+ * 1. Setup a signed_video_t session
+ * 2. Generate a golden SEI
+ * 3. Verify the golden SEI
+ */
+START_TEST(golden_sei_created)
+{
+
+  SignedVideoCodec codec = settings[_i].codec;
+  SignedVideoReturnCode sv_rc;
+  size_t sei_size = 0;
+  signed_video_t *sv = signed_video_create(codec);
+  ck_assert(sv);
+  char *private_key = NULL;
+  size_t private_key_size = 0;
+  // Setup the key
+  sv_rc = settings[_i].generate_key(NULL, &private_key, &private_key_size);
+  ck_assert_int_eq(sv_rc, SV_OK);
+
+  sv_rc = signed_video_set_private_key_new(sv, private_key, private_key_size);
+  ck_assert_int_eq(sv_rc, SV_OK);
+  sv_rc = signed_video_set_authenticity_level(sv, settings[_i].auth_level);
```

Comment:
Replace this line with setting the hash algo. That is relevant and is exercised in the test settings.